sdk/python/sawtooth_sdk/workload/sawtooth_workload.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+
+class SawtoothWorkload(object):
+    """
+       This is meant to be an abstract base class for all workloads.
```

Comment:
Is this the correct way to declare an abstract base class? I don't know the answer but the manage library does it differently using the abc module:
https://github.com/hyperledger/sawtooth-core/blob/master/manage/sawtooth_manage/node.py
https://docs.python.org/3.6/library/abc.html